docs/source/_templates/autosummary/class.rst

```diff
@@ -9,7 +9,9 @@
 
    .. autosummary::
    {% for item in methods %}
+   {%- if item not in inherited_members %}
```

Comment:
Does this include undocumented methods?

Reply:
This exclude members inherited from parent classes. It shouldn't change the logic of undoc members.